lib/Model/CMakeLists.txt

```diff
@@ -59,6 +59,18 @@ install(
     DESTINATION include/revng/Model
 )
 
+set(MODEL_JSONSCHEMA_PATH "${CMAKE_BINARY_DIR}/jsonschema.yml")
+tuple_tree_generator_jsonschema_from_headers(
+  "${MODEL_HEADERS}"
+  TUPLE-TREE-YAML
+  model
+  Binary
+  "Identifier"
+  "MetaAddress"
+  "${MODEL_JSONSCHEMA_PATH}"
+)
+add_custom_target(generated-model-jsonschema DEPENDS "${MODEL_JSONSCHEMA_PATH}")
+
```

Comment:
Because the model jsonschema is not needed before the commit that generated the python model from it, so I opted to include the change in `Generation from model jsonschema` rather than `Support jsonschema emission` (I'm using commit descriptions and not hashes since I'm force pushing)